resources/lang/pt_BR/cachet.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+return [
+    'support' => [
+        'section_heading' => 'Apoiar o Cachet',
+        'consider_supporting' => 'Por favor, considere apoiar o Cachet via *GitHub Sponsors*.',
+        'keep_up_to_date' => 'Mantenha-se atualizado com as últimas notícias e lançamentos seguindo o *blog do Cachet*.',
+        'work_in_progress_text' => 'O Cachet está em desenvolvimento ativo. As coisas ainda estão sujeitas a mudanças.',
+    ],
+    'powered_by' => 'Powered by',
+    'open_source_status_page' => 'The open-source status page.',
```

Comment:
left this as is. Could not find where it's being used.

Reply:
My bad! The string is used in https://github.com/cachethq/core/blob/main/resources/views/components/footer.blade.php#L6 but I made a typo. 

Fixed in https://github.com/cachethq/core/pull/172